packages/eventual-send/src/E.js

```diff
@@ -302,26 +302,90 @@ export default makeE;
  * `Promise<T>` or `T` itself.
  */
 
+/**
+ * A type that doesn't assume its parameter is local, but is satisfied with both
+ * local and remote references. It accepts both near and marshalled references
+ * that were returned from `Remotable` or `Far`.
+ *
+ * @template Primary The type of the primary reference.
+ * @template [Local=DataOnly<Primary>] The local properties of the object.
+ * @typedef {Primary | RemotableBrand<Local, Primary>} Remote
+ */
+
+/**
+ * A type that accepts either resolved or promised references that may be either
+ * near or marshalled. @see {ERef} and @see {Remote}.
+ *
+ * @template Primary The type of the primary reference.
+ * @template [Local=DataOnly<Primary>] The local properties of the object.
+ * @typedef {ERef<Remote<Primary, Local>>} ERemote
```

Comment:
How does this differ from `PassableCap`, `PassByReference`, or your `RemotableObject | RemotableBrand` pattern?

Having so many ways to say essentially the same thing makes me feel helpless. When next I need to write code, I don't know that I can do better than guess which type I should use. Not good.

Reply:
With `Remote` and `ERemote` gone, resolving.